packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -210,12 +210,28 @@ export type AllowedActions =
   | NotificationServicesPushControllerUpdateTriggerPushNotifications;
 
 // Events
-export type NotificationServicesControllerMessengerEvents =
+export type NotificationServicesControllerChangeEvent =
   ControllerStateChangeEvent<
     typeof controllerName,
     NotificationServicesControllerState
   >;
 
+export type MetamaskNotificationsControllerNotificationsListUpdatedEvent = {
+  type: `${typeof controllerName}:notificationsListUpdated`;
+  payload: [INotification[]];
+};
+
+export type MetamaskNotificationsControllerMarkNotificationsAsRead = {
+  type: `${typeof controllerName}:markNotificationsAsRead`;
+  payload: [INotification[]];
+};
```

Comment:
General observation on namespaced type names in the notifications controllers: 

If `Actions` and `Events` are safe to use because they'll always be imported under the `NotificationServicesController` namespace, isn't it redundant to specify `MetaMaskNotificationsController` in these types as well?

i.e. when imported, wouldn't the second option be preferable to the first?
```ts
NotificationServicesController.MetaMaskNotificationsControllerNotificationsListUpdatedEvent
NotificationServicesController.NotificationsListUpdatedEvent
```

As an aside, has the team considered organizing exports using the package.json "exports" field? i.e. 
```ts
import { ... } from '@metamask/notification-services-controller/notification-services-push-controller'
```
In this case, globally unique type names would need to be enforced.

Reply:
Hey @MajorLift 

> If Actions and Events are safe to use because they'll always be imported under the NotificationServicesController namespace, isn't it redundant to specify MetaMaskNotificationsController in these types as well?

Good catch, these were copied from the older controllers in the extension codebase. I'm making more changes & PRs so will add them in those :)

> As an aside, has the team considered organizing exports using the package.json "exports" field? 

YES, I'm glad you spotted this :D. Ideally we wanted to have multiple exports but we were unable to get this working in the `package.json`, so (for velocity) opted for named exports. We will be revisting this!